be/src/formats/parquet/file_reader.cpp

```diff
@@ -78,7 +78,7 @@ Status FileReader::init(HdfsScannerContext* ctx) {
     _scanner_ctx = ctx;
 #ifdef WITH_STARCACHE
     // Only support file metacache in starcache engine
-    if (ctx->use_file_metacache && config::datacache_enable) {
```

Comment:
why remove that?

Reply:
This condition judgement is redundant, see https://github.com/StarRocks/starrocks/blob/81424c0be009fa8ca8b9d500ccd517f82a3aa4ee/be/src/connector/hive_connector.cpp#L164-L167.